components/base/fake/fake.go

```diff
@@ -0,0 +1,160 @@
+// Package fake implements a fake base.
+package fake
+
+import (
+	"context"
+
+	"github.com/edaniels/golog"
+	"github.com/golang/geo/r3"
+	"github.com/pkg/errors"
+
+	"go.viam.com/rdk/components/base"
+	limo "go.viam.com/rdk/components/base/agilex"
+	"go.viam.com/rdk/components/base/boat"
+	"go.viam.com/rdk/components/base/wheeled"
+	"go.viam.com/rdk/components/generic"
+	"go.viam.com/rdk/config"
+	"go.viam.com/rdk/referenceframe"
+	"go.viam.com/rdk/registry"
+	"go.viam.com/rdk/resource"
+	"go.viam.com/rdk/services/slam"
+	"go.viam.com/rdk/spatialmath"
+)
+
+// Config is used for converting config attributes.
+type Config struct {
+	BaseModel string `json:"base-model,omitempty"`
```

Comment:
[major] I'd rather have the default be wheeled base right now without configuration parameters at all exposed to the user.

Reply:
It seems to be using `FrameConfig` anyway. We don't need to add those other bases, they're experimental and likely will have to use SetVelocity or SetPower. Let's keep work as narrow as possible.

Reply:
So hard code it as just being a wheeledBase always, and delete the Config struct entirely? I'm good with this just wanted to check this is what you are suggesting

Reply:
yep